nft-client/src/main.rs

```diff
@@ -0,0 +1,635 @@
+    },
+    #[structopt(name = "transfer", help = "Transfer one NFT to another address.")]
+    Transfer {
+        #[structopt(long = "contract", help = "NFT contract address")]
+        contract:     ContractAddressWrapper,
+        #[structopt(
+            name = "sender",
+            long = "sender",
+            help = "JSON file containing the account keys."
+        )]
+        sender:       PathBuf,
+        #[structopt(
+            name = "token_id",
+            long = "token",
+            help = "Token ID to transfer in the contract."
+        )]
+        token_ids:    Vec<cts1::TokenIdVec>,
+        #[structopt(
+            name = "from",
+            long = "from",
+            help = "The current owner of the token being transferred."
+        )]
+        from:         AddressWrapper,
+        #[structopt(
+            name = "to",
+            long = "to",
+            help = "The address to receive the token being transferred."
+        )]
+        to:           AddressWrapper,
+        #[structopt(
+            name = "to-func",
+            long = "to-func",
+            help = "The receive function name to call on the contract receiving tokens, only used \
+                    when `--to` is a contract address."
+        )]
+        to_func:      Option<OwnedReceiveNameWrapper>,
+        #[structopt(
+            name = "to-func-data",
+            long = "to-func-data",
+            help = "Some bytes (hex encoded) to include when calling receive function on the \
+                    contract receiving tokens, only used when `--to` is a contract address.",
+            default_value = ""
+        )]
+        to_func_data: cts1::ReceiveHookData,
+    },
+}
+
+#[tokio::main(flavor = "multi_thread")]
+async fn main() -> anyhow::Result<()> {
+    let app = {
+        let app = App::clap()
+            .setting(AppSettings::ArgRequiredElseHelp)
+            .global_setting(AppSettings::ColoredHelp);
+        let matches = app.get_matches();
+        App::from_clap(&matches)
+    };
+
+    match app.command {
+        Command::PrintState { contract } => {
+            // The GRPC client for Concordium node.
+            let mut grpc_client = Client::connect(app.endpoint, "rpcadmin".to_string())
+                .await
+                .context("Failed to connect to Node GRPC")?;
+
+            let consensus_status = grpc_client.get_consensus_status().await?;
+            let instance_info = grpc_client
+                .get_instance_info(contract.0, &consensus_status.best_block)
+                .await?;
+            let mut cursor = concordium_contracts_common::Cursor::new(instance_info.model);
+            let state = NFTContractState::deserial(&mut cursor)
+                .map_err(|_| anyhow!("Failed parsing contract state"))?;
+            pretty_print_contract_state(&state);
+        }
+        Command::Trace { contract, config } => {
+            println!("Connecting to PostgreSQL");
```

Comment:
I think these prints should probably be on stderr, and only output the user might want to save to a file should be on stdout.